### PR TITLE
Add option to disable schema request binding

### PR DESCRIPTION
**Pull request by leplatrem**

### CODE REVIEW

File: cornice/tests/test_schemas.py [hunk 1]
```diff
@@ -204,6 +204,18 @@ def test_colander_bound_schema_rebinds_to_new_request(self):
             field = the_schema.get_attributes(request=other_dummy_request)[3]
             self.assertEqual(field.validator.choices, ['c', 'd'])
 
+        def test_colander_request_binding_can_be_disabled(self):
+            dummy_request = {'x-foo': 'version_a'}
+            the_schema = CorniceSchema.from_colander(ToBoundSchema)
+            field = the_schema.get_attributes(request=dummy_request)[3]
+            # Deferred are resolved
```
Comment (ametaireau) [inline cornice/tests/test_schemas.py]: Should this be two tests?

**Comment by ametaireau** [comment]
We should add this to the documentation also, otherwise this looks good to me.

File: cornice/tests/test_schemas.py (hunk 1 above)
Comment (leplatrem) [inline cornice/tests/test_schemas.py]: +1

**Comment by leplatrem** [comment]
I wanted to mention it in the documentation, but found not appropriate existing section :/

**Comment by ametaireau** [comment]
There is no specific docs about the colander request binding yet, but we could add it to http://cornice.readthedocs.org/en/latest/validation.html#using-colander

**Comment by ametaireau** [comment]
Alternatively, adding a docstring could make this more explicit

**Comment by leplatrem** [comment]
I chose to rewrite this part of the documentation :)

**Comment by ametaireau** [comment]
Thanks!

**Pull request merged**